src/renderer/components/data-settings/data-settings.js

```diff
@@ -69,7 +70,7 @@ export default defineComponent({
       ]
     },
     primaryProfile: function () {
-      return JSON.parse(JSON.stringify(this.profileList[0]))
+      return deepCopy(this.profileList[0])
```

Comment:
I updated all references of `JSON.parse(JSON.stringify` to use deepCopy to make the code more consistent